### ISSUE TITLE
In game is wrong

### ISSUE THREAD

**Issue by wsthilaire** [RCVolus/league-prod-toolkit#81]
Few things, first league updated their in game scoreboard and the in game tab and team games need to be updated with positions and color. The scoreboard now shows turrets natively so that can be removed. Third the level up seems bugged and doesn’t show the color behind the number instead it just shows the black text number.

Thank you for making this free and public, your work is very appreciated!

**Comment by sdomi4** [comment]
Hi there :)
We have already fixed the overlay issues coming from the spectator changes in the spring-cleanup branch, but have not merged the changes yet since it also contains some structural changes that aren't finished. A workaround is below.

As for the color, could you make sure you don't have any custom CSS in your OBS browser source? The default CSS included with every browser source likes to eat colors and placements.

## Workaround for spectator changes:
- Download the updated modules for [ingame ](https://github.com/rcv-prod-toolkit/module-league-in-game) and [teams](https://github.com/rcv-prod-toolkit/module-teams)
- Delete the contents of the folders `...\league-prod-toolkit-develop\modules\league-in-game` and `...\league-prod-toolkit-develop\modules\rcv-teams`
- Copy the contents from the downloaded modules into their respective folders
- edit config.json in `...\league-prod-toolkit-develop\modules\config` and rename the `league-in-game` object to `module-league-ingame`:
```
"module-league-in-game": {
    "items": [],
    "level": []
  }
```
- Open PowerShell and navigate to the toolkit (or shift-rightclick in the toolkit directory to open PowerShell directly)
- Run `npm run build:modules`
- Run `npm start` or start.bat

**Note: the ingame webpage contained a typo, it has only been fixed in the file, but not the link in the ingame module. The page can be found at http://localhost:3003/pages/op-league-in-game/gfx/ingame.html**

**Comment by YI-YEE** [comment]
> league-prod-toolkit-core@0.0.1 start
> npm run build && node --trace-warnings dist/core/app.js


> league-prod-toolkit-core@0.0.1 build
> tsc && webpack

asset main-bundle.js 29 KiB [compared for emit] (name: main)
asset rcv.png 26.1 KiB [compared for emit] [from: frontend/rcv.png] [copied]
asset frontend-lib.ts 6.54 KiB [compared for emit] [from: frontend/frontend-lib.ts] [copied]
asset style.css 310 bytes [compared for emit] [from: frontend/style.css] [copied]
./frontend/frontend-lib.ts 7 KiB [built] [code generated]
./core/eventbus/LPTE.ts 564 bytes [built] [code generated]
webpack 5.38.1 compiled successfully in 4147 ms

> league-prod-toolkit-core@0.0.1 postbuild
> copyfiles core/web/views/**.* core/web/public/**.* dist

2022-03-23T14:36:47.783Z [info ] main        :  _          _       _____           _ _    _ _
2022-03-23T14:36:47.785Z [info ] main        : | |    ___ | |     |_   _|__   ___ | | | _(_) |_
2022-03-23T14:36:47.786Z [info ] main        : | |   / _ \| |       | |/ _ \ / _ \| | |/ / | __|
2022-03-23T14:36:47.786Z [info ] main        : | |__| (_) | |___    | | (_) | (_) | |   <| | |_
2022-03-23T14:36:47.786Z [info ] main        : |_____\___/|_____|   |_|\___/ \___/|_|_|\_\_|\__|
2022-03-23T14:36:47.787Z [info ] main        :
2022-03-23T14:36:47.787Z [info ] lpte-svc    : Initialized event bus.
2022-03-23T14:36:47.789Z [info ] module-svc  : Initializing module service.
2022-03-23T14:36:47.824Z [info ] module-svc  : Initialized 19 module(s). Now loading plugins.
2022-03-23T14:36:47.825Z [info ] module-svc  : Loaded 19 plugin(s).
2022-03-23T14:36:50.806Z [info ] main        : LoL Toolkit started up successfully.
2022-03-23T14:36:50.810Z [info ] server      : Listening for requests on http://localhost:3003
2022-03-23T14:36:50.812Z [info ] module-svc  : Plugin status changed: plugin=layout-volu-europe, old=UNAVAILABLE, new=RUNNING
2022-03-23T14:36:50.813Z [info ] module-svc  : Plugin status changed: plugin=config, old=UNAVAILABLE, new=RUNNING
2022-03-23T14:36:50.815Z [info ] module-svc  : Plugin status changed: plugin=example-module, old=UNAVAILABLE, new=RUNNING
2022-03-23T14:36:50.816Z [info ] module-svc  : Plugin status changed: plugin=league-end-of-game, old=UNAVAILABLE, new=RUNNING
2022-03-23T14:36:50.818Z [info ] module-svc  : Plugin status changed: plugin=module-teams, old=UNAVAILABLE, new=RUNNING
2022-03-23T14:36:50.819Z [info ] module-svc  : Plugin status changed: plugin=prod-clock, old=UNAVAILABLE, new=RUNNING
2022-03-23T14:36:50.820Z [info ] module-svc  : Plugin status changed: plugin=provider-webapi, old=UNAVAILABLE, new=RUNNING
2022-03-23T14:36:50.822Z [info ] module-svc  : Plugin status changed: plugin=rcv-caster, old=UNAVAILABLE, new=RUNNING
2022-03-23T14:36:50.823Z [info ] module-svc  : Plugin status changed: plugin=rcv-gfx, old=UNAVAILABLE, new=RUNNING
2022-03-23T14:36:50.824Z [info ] module-svc  : Plugin status changed: plugin=rcv-rune-gfx, old=UNAVAILABLE, new=RUNNING
2022-03-23T14:36:50.825Z [info ] module-svc  : Plugin status changed: plugin=rcv-tournament-tree, old=UNAVAILABLE, new=RUNNING
2022-03-23T14:36:50.826Z [info ] module-svc  : Plugin status changed: plugin=state-league, old=UNAVAILABLE, new=RUNNING
2022-03-23T14:36:50.829Z [info ] module-svc  : Plugin status changed: plugin=valorant-pregame, old=UNAVAILABLE, new=RUNNING
2022-03-23T14:36:50.830Z [info ] module-svc  : Plugin status changed: plugin=valorant-state, old=UNAVAILABLE, new=RUNNING
2022-03-23T14:36:50.846Z [info ] module-svc  : Plugin status changed: plugin=database, old=UNAVAILABLE, new=RUNNING
2022-03-23T14:36:50.847Z [info ] module-svc  : Plugin status changed: plugin=module-league-in-game, old=UNAVAILABLE, new=RUNNING
2022-03-23T14:36:50.858Z [info ] module-svc  : Plugin status changed: plugin=themeing, old=UNAVAILABLE, new=RUNNING
2022-03-23T14:36:51.123Z [info ] module-svc  : Plugin status changed: plugin=static-league, old=UNAVAILABLE, new=RUNNING
2022-03-23T14:36:51.808Z [info ] module-svc  : Plugin status changed: plugin=valorant-static, old=UNAVAILABLE, new=RUNNING
setting storageRoot to F:\Observer Tool\league-prod-toolkit-develop\node_modules\leaguejs\StaticDataDefaultRoot
2022-03-23T14:38:38.914Z [warn ] lpte-svc    : Request was sent, but no handler was executed. This will result in a timeout. Meta={"namespace":"league-in-game","type":"request","version":1,"reply":"request-c0cc2431","channelType":"REQUEST"}
2022-03-23T14:42:42.368Z [warn ] lpte-svc    : Request was sent, but no handler was executed. This will result in a timeout. Meta={"namespace":"league-in-game","type":"request","version":1,"reply":"request-f70c44d1","channelType":"REQUEST"}
2022-03-23T14:44:10.736Z [warn ] lpte-svc    : Request was sent, but no handler was executed. This will result in a timeout. Meta={"namespace":"league-in-game","type":"request","version":1,"reply":"request-f832c5d0","channelType":"REQUEST"}
2022-03-23T14:50:48.073Z [warn ] lpte-svc    : Request was sent, but no handler was executed. This will result in a timeout. Meta={"namespace":"league-in-game","type":"request","version":1,"reply":"request-1d95c1df","channelType":"REQUEST"}
2022-03-23T14:52:43.833Z [warn ] lpte-svc    : Request was sent, but no handler was executed. This will result in a timeout. Meta={"namespace":"league-in-game","type":"request","version":1,"reply":"request-355723b7","channelType":"REQUEST"}
2022-03-23T14:52:48.435Z [info ] plugin-config: config for module-league-in-game saved!
2022-03-23T14:55:04.792Z [warn ] lpte-svc    : Request was sent, but no handler was executed. This will result in a timeout. Meta={"namespace":"league-in-game","type":"request","version":1,"reply":"request-63a403a1","channelType":"REQUEST"}


I got this problem...

**Comment by sdomi4** [comment]
resolved with release 0.9, updating by hand is no longer necessary
